doc/sphinx-guides/source/api/native-api.rst

```diff
@@ -526,6 +526,39 @@ To create a dataset, you must supply a JSON file that contains at least the foll
 - Description Text
 - Subject
 
+Alternatively, you can turn off the validation of the dataset by using the optional "doNotValidate" parameter. This parameter works only when the ":AllowInvalidMetadataThroughAPI" setting is set to "ture". Your solr configuration must be up-to-date with the "schema.xml" file for this feature to work.
```

Comment:
typo here "ture"

Reply:
Thanks! Typo is fixed. I have also merged the current develop branch into this pull request branch.